File: src/main/java/me/kingtux/kingserverinfo/events/LeaveEvent.java
package me.kingtux.kingserverinfo.events;

import java.util.List;
import me.kingtux.kingserverinfo.KingServerInfoMain;
import org.bukkit.Bukkit;
import org.bukkit.ChatColor;
import org.bukkit.entity.Player;
import org.bukkit.event.EventHandler;
import org.bukkit.event.Listener;
import org.bukkit.event.player.PlayerQuitEvent;

public class LeaveEvent implements Listener {
    private KingServerInfoMain plugin;

    public LeaveEvent(KingServerInfoMain pl) {
        plugin = pl;
    }

    @EventHandler
    public void playerLeaveEvent(PlayerQuitEvent e) {
        Player player = e.getPlayer();
        List<String> leaveMessage = plugin.getConfigSettings().getOnLeaveMessage();
        for (String s : leaveMessage) {
            s = s.replace("{player}", player.getName());
            Bukkit.getServer().broadcastMessage(
                    ChatColor.translateAlternateColorCodes('&',
                            plugin.getConfigSettings().getPrefix() + " " + s));
        }
    }
}

File: src/me/kingtux/kingserverinfo/events/LeaveEvent.kt
package me.kingtux.kingserverinfo.events

import me.kingtux.kingserverinfo.KingServerInfoMain
import org.bukkit.Bukkit
import org.bukkit.ChatColor
import org.bukkit.event.EventHandler
import org.bukkit.event.Listener
import org.bukkit.event.player.PlayerQuitEvent

class LeaveEvent(private val plugin: KingServerInfoMain) : Listener {

    @EventHandler
    fun playerLeaveEvent(e: PlayerQuitEvent) {
        val player = e.player
        var leaveMessage = plugin.configSettings!!.onLeaveMessage
        for (s in leaveMessage) {
            var s = s.replace("{player}", player.name)
            Bukkit.getServer().broadcastMessage(
                    ChatColor.translateAlternateColorCodes('&',
                            plugin.configSettings!!.prefix + " " + s))
        }
    }
}
